src/nginxconfig/generators/conf/nginx.conf.js

```diff
@@ -73,8 +73,54 @@ export default (domains, global) => {
     config.http.push(['include', 'mime.types']);
     config.http.push(['default_type', 'application/octet-stream']);
 
+    // Define default log format as an array
+    let logging = ['$remote_addr', '-', '$remote_user', '[$time_local]',
+    '"$request"', '$status', '$body_bytes_sent',
+    '"$http_referer"', '"$http_user_agent"'];
+
+    // Append Cloudflare request headers to the default log format
+    if (global.logging.cloudflare.computed) {
+        config.http.push(['# Log Format', '']);
+
+        if (global.logging.cfRay.computed) {
+            logging.push('$http_cf_ray');
+        }
```

Comment:
```suggestion
        if (global.logging.cfRay.computed)
            logging.push('$http_cf_ray');
```
We don't need curlys here